lib/db.js

```diff
@@ -724,6 +750,16 @@ module.exports = function (
     return this.pool.post('/account/' + account.uid.toString('hex') + '/verifyEmail')
   }
 
+  DB.prototype.verifyToken = function (tokenVerificationId, accountData) {
+    log.trace({ op: 'DB.verifyToken', tokenVerificationId: tokenVerificationId })
+    return this.pool.post('/token/' + tokenVerificationId.toString('hex') + '/verify',
+        unbuffer(
+          accountData,
+          'inplace'
+        )
+    )
+  }
+
```

Comment:
I believe this has now reverted to being "verifyTokens" plural, as of https://github.com/mozilla/fxa-auth-db-mysql/commit/5a45e40e90edcf261be0e9b024e2fedc432e7e91